guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -610,6 +610,8 @@ class View extends Template
 
 6.4.4.12. Any customizations to the domain/business logic MUST be executed on the Service Contracts layer, and so MUST be declared in the `global` area of configuration.
 
+6.4.4.13. Service contracts MUST NOT rely on the execution context (application area). Services implementation MUST NOT know about the application state.
```

Comment:
Should it be SHOULD or can it be applied in all cases today?

Reply:
for all cases, for all service contracts